factory/pom.xml

```diff
@@ -19,19 +18,24 @@
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
+  <parent>
+    <groupId>org.sonatype.oss</groupId>
+    <artifactId>oss-parent</artifactId>
```

Comment:
I don't think that `factory` uses this yet except in its integration tests. Are we able to avoid doing so here, even if it requires pulling more plugin configuration into the aggregator or `annotations` build (like what you already have in `processor`—and see my comment about whether `processor` should have aggregator as its parent)?

I say this because we've had [trouble with `oss-parent`](https://github.com/google/truth/pull/1043) in the past. (It's deprecated, too, probably to reflect that it's not being kept up to date with plugin versions.)